Sources/Model/Paging/HyperwalletQueryParam.swift

```diff
@@ -18,25 +18,33 @@
 
 import Foundation
 
-/// Representation of the common pagination fields
-public class HyperwalletPagination {
+/// Representation of the common query parameters.
+public class HyperwalletQueryParam {
     /// The maximum number of records that will be returned per page
     public var limit: Int
     /// The number of records to skip. If no filters are applied, records will be skipped from the beginning
     /// (based on default sort criteria). Default value is 0. Range is from 0 to {n-1} where n = number of
     /// matching records for the query.
     public var offset: Int
 
-    /// Creates a new instance of `HyperwalletPagination`]
+    /// Creates a new instance of `HyperwalletQueryParam`]
     public init() {
         limit = 10
         offset = 0
     }
 
-    /// Builds the `HyperwalletTransferMethodPagination`'s URL Queries
+    private enum QueryParam: String {
+        case limit
+        case offset
+    }
+
+    /// Builds the `HyperwalletTransferMethodQueryParam`'s URL Queries
```

Comment:
Remove HyperwalletQueryParam from the comment.

Reply:
fixed